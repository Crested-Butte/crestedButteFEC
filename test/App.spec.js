/**
 * @jest-environment jsdom
 */

import React from 'react';
import renderer from 'react-test-renderer';
import $ from 'jquery';
import App from '../src/App.js'




describe('App', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<App
    />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});