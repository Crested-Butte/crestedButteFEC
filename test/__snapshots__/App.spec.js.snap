// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders 1`] = `
<div>
  <div
    className="header"
  >
    <div
      className="logo"
    >
      <img
        src="./data/logo.png"
      />
    </div>
    <div
      className="title"
    >
      <h1>
        CRESTED BUTTE
      </h1>
    </div>
    <div
      className="shopping-cart"
    >
      <i
        className="fas fa-shopping-cart"
      />
      <div
        className="display"
      >
         
      </div>
    </div>
    <div
      className="clear"
    />
  </div>
  <div
    className="sub-header header"
  >
    <div
      className="logo"
    >
      <img
        src="./data/darkLogo.png"
      />
    </div>
    <div
      className="title"
    >
      <h1>
        CRESTED BUTTE
      </h1>
    </div>
    <div
      className="shopping-cart"
    >
      <i
        className="fas fa-shopping-cart"
      />
      <div
        className="display"
      >
         
      </div>
    </div>
    <div
      className="clear"
    />
  </div>
  <div
    className="product-details"
  >
    <div>
      loading
    </div>
  </div>
  <div
    className="related-items"
  >
    <div>
      loading
    </div>
  </div>
  <div
    className="questions-answers"
  >
    <div>
      loading
    </div>
  </div>
  <div
    className="ratings-reviews"
  >
    <div>
      loading
    </div>
  </div>
  <button
    className="to-top display"
    onClick={[Function]}
  >
    <i
      className="fas fa-arrow-up"
    />
  </button>
  <div
    className="footer"
  >
    <span
      className="terms"
    >
      Terms of Service
    </span>
     
    <span
      className="privacy"
    >
       Privacy Policy
    </span>
     
    <span
      className="copyright"
    >
      © 2022 Crested Butte, Inc.
    </span>
  </div>
</div>
`;
